.github/workflows/tests.yml

```diff
@@ -18,8 +18,8 @@ jobs: # Docs: <https://git.io/JvxXE>
     strategy:
       fail-fast: false
       matrix:
-        setup: [basic, lowest]
-        php: ['7.3', '7.4', '8.0']
+        setup: [ 'basic', 'lowest' ]
```

Comment:
Don't forget to remove the argument `--no-suggest`

Reply:
Removed